mods/ra/maps/allies-02/allies02.lua

```diff
@@ -220,10 +222,10 @@ WorldLoaded = function()
 
 	InitObjectives(player)
 
-	ussrObj = ussr.AddObjective("Deny the allies!")
+	ussrObj = AddPrimaryObjective(ussr, "deny-allies")
```

Comment:
Most missions have an enemy objective like this. The player will never see it so translation not needed.